app/Http/Controllers/PhotoEditorController.php

```diff
@@ -45,13 +49,29 @@ public function rotate(Request $request)
 			return false;
```

Comment:
false needs to be a string.
```suggestion
			return 'false';
```